specs/tendrl_ansible_grafana_installation.adoc

```diff
@@ -0,0 +1,156 @@
+= Tendrl ansible to install grafana
+
+Needs ansible playbook scripts for automatic installation and configuration of
+grafana and tendrl metrics in the tendrl server.
+
+
+== Problem description
+
+Grafana will be used in tendrl to trigger alerts and visualize the tendrl
+metrics. Currently, Tendrl-ansible is used to install and configure tendrl
+packages. It should also support installation and configuration of tendrl
+metrics and grafana package installation.
+This specification provides the required changes in tendrl-ansible tool to
+support installation and configuration of tendrl metrics and grafana.
+
+
+== Use Cases
+
+* Provide playbook for tendrl-ansible to install tendrl-metrics and grafana
+along with server installation.
+
+
+== Proposed change
+
+* Add the repo to install latest grafana and its dependencies
+
+* Install tendrl metrics, grafana, fontconfig, freetype* and urw-fonts
+
+* Configure tendrl metrics and  grafana
+  This should add the required environment variables, set log path, database
```

Comment:
Sure, I will add the details as bellow:
grafana_config:
grafana_database:
   type: graphite?
   host: <ip>:<port>
   name: grafana
We can add credentials also if required for grafana. @anmolbabu, please provide your suggestion for tendrl-graphite port details.
   user: root
   password: ""

# Server Configuration Details:
grafana_server:
  protocol: http
  # The ip address to bind to, empty will bind to all interfaces
  http_addr: ""
  http_port: 3000
  domain: <public ip to access grafana from a browser>
  root_url: "%(protocol)s://%(domain)s:%(http_port)s/"
# binary where the static  files are placed, public: html/js/css
   static_root_path: public
# SSL cert & key file if grafana_server_protocol is https
  cert_file: ""
  cert_key: ""

grafana_session:
# Session provider could be either memory, file mysql. The default is "memory"
   grafana_session_provider: file
# Session life time, default is 86400
    session_life_time: 86400

Also we can configure grafana user, admin user credentials.